src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_up_commands.py

```diff
@@ -295,6 +295,76 @@ def test_webapp_up_dotnet6_e2e(self, resource_group):
         import shutil
         shutil.rmtree(temp_dir)
 
+    @live_only()
```

Comment:
May I ask why this test is `live_only()`?

Reply:
All of the `az webapp up` tests are live only for the reasons we mentioned here -- https://github.com/Azure/azure-cli/pull/19344#discussion_r699539541